frontend/src/pages/ArtifactDetails.tsx

```diff
@@ -14,38 +14,56 @@
  * limitations under the License.
  */
 
-import * as React from 'react';
-import {Page} from './Page';
-import {ToolbarProps} from '../components/Toolbar';
-import {RoutePage, RouteParams} from '../components/Router';
-import {Api, ArtifactProperties} from '../lib/Api';
-import {MlMetadataArtifact} from '../apis/service';
-import {classes} from 'typestyle';
-import {commonCss, padding} from '../Css';
-import {CircularProgress} from '@material-ui/core';
-import {titleCase, getResourceProperty} from '../lib/Utils';
-import {ResourceInfo} from '../components/ResourceInfo';
+import * as React from "react";
+import { Page } from "./Page";
+import { ToolbarProps } from "../components/Toolbar";
+import { RoutePage, RouteParams } from "../components/Router";
+import { Api, ArtifactProperties } from "../lib/Api";
+import { MlMetadataArtifact } from "../apis/service";
+import { classes } from "typestyle";
+import { commonCss, padding } from "../Css";
+import { CircularProgress } from "@material-ui/core";
+import { titleCase, getResourceProperty } from "../lib/Utils";
+import { ResourceInfo } from "../components/ResourceInfo";
+import MD2Tabs from "../atoms/MD2Tabs";
+
+export enum ArtifactDetailsTab {
+  OVERVIEW = 0,
+  LINEAGE_EXPLORER = 1,
+  DEPLOYMENTS = 2
+}
+
+const tabs = {
+  [ArtifactDetailsTab.OVERVIEW]: { name: "Overview" },
+  [ArtifactDetailsTab.LINEAGE_EXPLORER]: { name: "Lineage Explorer" },
+  [ArtifactDetailsTab.DEPLOYMENTS]: { name: "Deployments" }
+};
+
+const tabNames = Object.values(tabs).map(tabConfig => tabConfig.name);
 
 interface ArtifactDetailsState {
   artifact?: MlMetadataArtifact;
+  selectedTab: ArtifactDetailsTab;
 }
 
 export default class ArtifactDetails extends Page<{}, ArtifactDetailsState> {
   private api = Api.getInstance();
 
   constructor(props: {}) {
     super(props);
-    this.state = {};
+    this.state = {
+      selectedTab: ArtifactDetailsTab.OVERVIEW
+    };
     this.load = this.load.bind(this);
   }
 
   private get fullTypeName(): string {
-    return this.props.match.params[RouteParams.ARTIFACT_TYPE] || '';
+    return this.props.match.params[RouteParams.ARTIFACT_TYPE] || "";
```

Comment:
The preferred JS convention is single-quotes, we use the same practice in Kubeflow/kubeflow
```suggestion
    return this.props.match.params[RouteParams.ARTIFACT_TYPE] || '';
```

Reply:
Done.